### ISSUE TITLE
[BUG] [Formatter] Incorrect formatting on function index

### ISSUE THREAD

**Issue by nrnvgh** [Riverside-Healthcare/djLint#693]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: OSX 13.2.1
 - Python Version: 3.10.11, 3.11.3
 - djLint Version 1.31.0
 - template language: jinja

## Issue
Formatter inserts space when directly indexing a list returned by a function. To wit:

```
djlint --check --profile=jinja templates/list_element.j2

Checking 1/1 files ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 00:00


templates/list_element.j2
────────────────────────────────────────────────────
@@ -1,6 +1,5 @@

 {% set item_list = ["1", "4", "9"] %}
 * Item: {{ item_list[2] }}
-
 {% set item = item_list[2] + "/cidr_thing" %}
-New Val[1]: {{ item.split("/")[1] }}
-New Val.1: {{ item.split("/").1 }}
+New Val[1]: {{ item.split("/") [1] }}  <== This
+New Val.1: {{ item.split("/") .1 }}.    <== And also this

1 file would be updated.
```
This doesn't break rendering, but djlint's behavior seem wrong to me; foo.function(args)[index] is, as far as I know, a valid construct.

## How To Reproduce
Script:
```python
#!/usr/bin/env python3

import sys
import argparse

import jinja2


def handle_arguments():
    parser = argparse.ArgumentParser()
    parser.add_argument('template')

    return parser.parse_args()


def main(args: argparse.Namespace):
    template = jinja2.Environment(
        loader=jinja2.FileSystemLoader("templates"),
        trim_blocks=True,
        lstrip_blocks=True,
    ).get_template(args.template)

    print(template.render())

if __name__ == '__main__':
    parsed_args = handle_arguments()

    sys.exit(main(parsed_args))
```

Template:
```
{% set item_list = ["1", "4", "9"] %}
* Item: {{ item_list[2] }}

{% set item = item_list[2] + "/cidr_thing" %}
New Val[1]: {{ item.split("/")[1] }}
New Val.1: {{ item.split("/").1 }}
```

**Comment by nrnvgh** [comment]
Confirmed on this end; thanks.
